lib/PlaidClient.js

```diff
@@ -79,13 +80,23 @@ Client.prototype._authenticatedRequest =
 Client.prototype.createPublicToken =
   requestWithAccessToken('/item/public_token/create', false);
 
-// (BETA) createItemAddToken(Function)
+// (BETA) createItemAddToken(CreateItemAddTokenOptions, Function)
 Client.prototype.createItemAddToken =
-  function(cb) {
-    return this._authenticatedRequest({
-      path: '/item/add_token/create',
-    }, cb);
-  };
+    function(options, cb) {
+        if (typeof options === 'function') {
+            console.warn('passing callback only to' +
+                ' createItemAddToken is deprecated');
+            cb = options;
+            options = {};
+        }
+        const {
+            user_identity,
+        } = options || {};
+        return this._authenticatedRequest({
+            path: '/item/add_token/create',
+            body: helpers.processUserFields(user_identity),
+        }, cb);
+    };
```

Comment:
this repo / file uses 2-space indentation

Reply:
⚡